docs/src/pages/guides/examples.md

```diff
@@ -214,3 +214,56 @@ probes:
 ```
 
 Using the above configuration, Monika will perform a login request in the first request, then use the returned token in the Authorization header of the second request.
+
+### Pass Response Data to Request Body
+
+Continuing with the examples from wwwh.reqres.in above, say we would like to use the previous GET request to perform a POST /login. If the initial data from reqres returned is something like below:
+
+```yaml
+{
+  'data':
+    {
+      'id': 1,
+      'email': 'george.bluth@reqres.in',
+      'first_name': 'George',
+      'last_name': 'Bluth',
+      'avatar': 'https://reqres.in/img/faces/1-image.jpg',
+    },
+  ....,
+}
+```
+
+Then your chain request to get the user body.email would look something like:
```

Comment:
Then you can use the user's email in the login request's `body` as follows

Reply:
fixed, thanks!